inlong-sort/sort-connectors/jdbc/src/main/java/org/apache/inlong/sort/jdbc/internal/JdbcBatchingOutputFormat.java

```diff
@@ -184,7 +191,7 @@ public void open(int taskNumber, int numTasks) throws IOException {
                                     if (!closed) {
                                         try {
                                             flush();
-                                            if (sinkMetricData != null) {
+                                            if (sinkMetricData != null && dirtySink == null) {
```

Comment:
Why metric data report dependent on 'dirtySink == null'?

Reply:
This report is only needed when TableMetricExecutor is not initialized, since TableMetricExecutor will do precise metric report

Reply:
Please add some description for the code

Reply:
added